### DISCUSSION TITLE
Not printing progress during execution on Jupyter notebook

### DISCUSSION THREAD

**Discussion by johnmarkacala** [ultralytics/yolov5#5817] [Q&A]
![image](https://user-images.githubusercontent.com/85044108/143817774-91eb389b-6437-4f62-bc3f-7e32f84a0f02.png)

I tried it on collab and it seems to print the progress just fine but not on my local Jupyter notebook. I'm not sure where to look for answers. Sorry for bothering and thanks

**Comment by glenn-jocher** [comment]
@johnmarkacala 👋 hi, thanks for letting us know about this possible problem with YOLOv5 🚀. We've created a few short guidelines below to help users provide what we need in order to get started investigating a possible problem.

## How to create a Minimal, Reproducible Example

When asking a question, people will be better able to provide help if you provide **code** that they can easily understand and use to **reproduce** the problem. This is referred to by community members as creating a [minimum reproducible example](https://stackoverflow.com/help/minimal-reproducible-example). Your code that reproduces the problem should be:

- ✅ **Minimal** – Use as little code as possible to produce the problem
- ✅ **Complete** – Provide all parts someone else needs to reproduce the problem
- ✅ **Reproducible** – Test the code you're about to provide to make sure it reproduces the problem

For [Ultralytics](https://ultralytics.com/) to provide assistance your code should also be:

- ✅ **Current** – Verify that your code is up-to-date with GitHub [master](https://github.com/ultralytics/yolov5/tree/master), and if necessary `git pull` or `git clone` a new copy to ensure your problem has not already been solved in [master](https://github.com/ultralytics/yolov5/tree/master).
- ✅ **Unmodified** – Your problem must be reproducible using official YOLOv5 code without changes. [Ultralytics](https://ultralytics.com/) does not provide support for custom code ⚠️.

If you believe your problem meets all the above criteria, please close this issue and raise a new one using the 🐛 **Bug Report** [template](https://github.com/ultralytics/yolov5/issues/new/choose) with a [minimum reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) to help us better understand and diagnose your problem. 

Thank you! 😃

**Comment by gboeer** [comment]
Hi, this can happen if the notebook is not trusted, i.e. javascript is disabled. 
Check if you mark it as trusted, in the upper right corner of the notebook.
